### PR TITLE
Ch setup up django project

### PR DESCRIPTION
**Pull request by PatrickMugayaJoel**
@bisonlou @kbjude @kalsmic @mwinel @AtamaZack @kabohaJeanMark

### CODE REVIEW

File: questioner/settings.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+WSGI_APPLICATION = 'questioner.wsgi.application'
+
+
+# Database
+# https://docs.djangoproject.com/en/2.1/ref/settings/#databases
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.sqlite3',
```
Comment (bisonlou) [inline questioner/settings.py:79]: I think we are going to be using postgresql

**Review by bisonlou — CHANGES_REQUESTED** [review]
Well done

File: questioner/settings.py (hunk 1 above)
Comment (kalsmic) [reply questioner/settings.py:79]: His initial code had covered this. @PatrickMugayaJoel let us have the database credentials picked from environmental variables.
```

DATABASES = {
    'default': {
        'ENGINE': 'django.db.backends.postgresql',
        'NAME':  os.environ.get('DB_NAME'),
        'USER':  os.environ.get('DB_USER'),
        'PASSWORD': os.environ.get('DB_PASSWORD'),
        'HOST':  os.environ.get('DB_HOST'),
        'PORT': '5432',
    }
}

```

Comment (ZackAtama) [reply questioner/settings.py:79]: Good progress.

Comment (kalsmic) [reply questioner/settings.py:79]: Let @bisonlou, do the postgres setup since he is doing the database setup

File: questioner/settings.py [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    'django.middleware.common.CommonMiddleware',
+    'django.middleware.csrf.CsrfViewMiddleware',
+    'django.contrib.auth.middleware.AuthenticationMiddleware',
+    'django.contrib.messages.middleware.MessageMiddleware',
+    'django.middleware.clickjacking.XFrameOptionsMiddleware',
+]
+
+ROOT_URLCONF = 'questioner.urls'
+
+TEMPLATES = [
```
Comment (mwinel) [inline questioner/settings.py:55]: Do we need this if we are just to implement API endpoints. I am presuming we are going to implement the front end in React.

**Comment by mwinel** [comment]
One thing I do not see is the api app that is meant to hold the API logic integrated into the project.

File: questioner/settings.py (hunk 2 above)
Comment (KabohaJeanMark) [reply questioner/settings.py:55]: Interesting point @mwinel. Perhaps we should have this for now and we will cross that bridge when we come to it. Or we can ask in advance?

File: questioner/settings.py (hunk 1 above)
Comment (KabohaJeanMark) [reply questioner/settings.py:79]: This is good progress indeed.

**Pull request closed without merging**